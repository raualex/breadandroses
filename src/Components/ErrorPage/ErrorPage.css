.error-container {
  background-color: rgba(243,243,243,0.8);
  border: solid 6px #000;
  border-radius: 48px;
  margin: 8px;
  padding: 20px;
}

.error-msg {
  font-family: 'Black Han Sans', sans-serif;
  margin: 26px;
}

.error-title {
  -webkit-text-stroke: 2px #000;
  color: #CC2B1D;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 2.5em;
}