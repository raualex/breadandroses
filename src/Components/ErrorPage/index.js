import React from 'react';
import './ErrorPage.css';

export const ErrorPage = () => {

  return (
    <div className="error-container">
      <h1 className="error-title">HALT</h1>
      <p className="error-msg">
        You are in error, comrade.  This page does not exist!
      </p>
    </div>
  )
}

export default ErrorPage;