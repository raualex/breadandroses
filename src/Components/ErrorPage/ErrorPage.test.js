import React from 'react';
import ErrorPage from './';
import { shallow } from 'enzyme';

describe('ErrorPage', () => {

  it('should match the snapshot', () => {
    
    let wrapper = shallow(<ErrorPage />)

    expect(wrapper).toMatchSnapshot()
  });
});