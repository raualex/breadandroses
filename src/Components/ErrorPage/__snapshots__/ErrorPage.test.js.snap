// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="error-title"
        >
          HALT
        </h1>,
        <p
          className="error-msg"
        >
          You are in error, comrade.  This page does not exist!
        </p>,
      ],
      "className": "error-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "HALT",
          "className": "error-title",
        },
        "ref": null,
        "rendered": "HALT",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "You are in error, comrade.  This page does not exist!",
          "className": "error-msg",
        },
        "ref": null,
        "rendered": "You are in error, comrade.  This page does not exist!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="error-title"
          >
            HALT
          </h1>,
          <p
            className="error-msg"
          >
            You are in error, comrade.  This page does not exist!
          </p>,
        ],
        "className": "error-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "HALT",
            "className": "error-title",
          },
          "ref": null,
          "rendered": "HALT",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "You are in error, comrade.  This page does not exist!",
            "className": "error-msg",
          },
          "ref": null,
          "rendered": "You are in error, comrade.  This page does not exist!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
